### PR TITLE
clean up age, updated_at

### PR DESCRIPTION
**Pull request by brendanashworth**
This commit cleans up the age and updated_at fields. It now uses dates
that are relevant to now instead of hours (i.e. "six days ago", "four
hours ago", "one month ago"). It also uses the correct dates now, as age
used to use the time between the created_at date and the updated_at
date. It now sorts the dates correctly as well.

Unrelated, it adds `go get` to the readme, which is required to build
the software.

It adds a new dependency, momentjs, a very handy date management tool to generate the humanized dates.

Fixes: https://github.com/minio/buzz/issues/57

### CODE REVIEW

File: github-pull-requests.go
Comment (deekoder) [inline github-pull-requests.go:94]: I think we want to compute : time elapsed since last updated i.e age = now - last_updated.

Comment (deekoder) [reply github-pull-requests.go:94]: And convert that delta to a human readable format.

**Review by deekoder — CHANGES_REQUESTED** [review]

File: github-pull-requests.go
Comment (brendanashworth) [reply github-pull-requests.go:94]: Yes — instead of doing it in Go it's done in the [browser](https://github.com/brendanashworth/buzz/blob/797abe4c8f55f8b2bd15bc8270a5789bf623f786/static/js/functions.js#L106) now. 

**edit:** removed a false comment

File: static/js/functions.js
Comment (ebozduman) [inline static/js/functions.js]: Is it possible to DRY this and line# 59? 
It is exactly the same code.

Comment (brendanashworth) [reply static/js/functions.js]: Yes! Thanks for catching that, I've fixed it in the latest version. PTAL

File: github-pull-requests.go
Comment (deekoder) [reply github-pull-requests.go:94]: ok i see that now.

**Review by deekoder — APPROVED** [review]

File: static/js/functions.js
Comment (ebozduman) [inline static/js/functions.js:95]: Don't you need to provide data and type parameters when calling `convertDate` method?

**Comment by deekoder** [comment]
waiting on @ebozduman  to approve these changes before merging.

**Comment by brendanashworth** [comment]
@deekoder, @ebozduman and I just cleared that up, it should be good to merge.

**Review by ebozduman — APPROVED** [review]

**Comment by brendanashworth** [comment]
Thanks!

**Pull request merged**
